floyd/cli/data.py

```diff
@@ -288,7 +289,10 @@ def delete(ids, yes):
 @click.argument('source')
 def add(source):
     """
-    Create data for current dataset from a given source, for example: foo/projects/bar/1/output
+    Add job output to a dataset.
+
+    This will create a new dataset version with the job output.
+    Use the full job name: foo/projects/bar/1/home or foo/projects/bar/1/output
```

Comment:
Do we explain why their files would be in `/home` anywhere in the docs? I know we mention it in the workspaces page, but not sure about the jobs.

It's also slightly confusing to say "Add job output" since you are also added any and all the "source code" for a job when you're doing this. Might try `Creates a new dataset version from the contents of a job run`

Reply:
I like your recommendation. 

As for '/home', agreed that it is not obvious. Ideally we should support referring to the job name without the '/home' suffix. I'll file a ticket to track that.